src/Form/AdminSettingsForm.php

```diff
@@ -93,14 +105,14 @@ private function preValidateData() {
       SettingsHelper::isInvalidCredentialSiteId($credential_settings['site_id']) ||
       SettingsHelper::isInvalidCredentialAssetsUrl($credential_settings['assets_url'])
     ) {
-      drupal_set_message(t('The Acquia Lift module requires a valid Account ID, Site ID, and Assets URL to complete activation.'), 'warning');
+      $this->messenger->addWarning(t('The Acquia Lift module requires a valid Account ID, Site ID, and Assets URL to complete activation.'));
```

Comment:
`drupal_set_message()` is obsoleted. Replace by `$this->messenger->addWarning()`.